verilog/tools/kythe/indexing_facts_tree_extractor.h

```diff
@@ -0,0 +1,83 @@
+// Copyright 2017-2020 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef VERIBLE_VERILOG_TOOLS_KYTHE_INDEXING_FACTS_TREE_EXTRACTOR_H_
+#define VERIBLE_VERILOG_TOOLS_KYTHE_INDEXING_FACTS_TREE_EXTRACTOR_H_
+
+#include "absl/strings/string_view.h"
+#include "common/text/tree_context_visitor.h"
+#include "common/text/tree_utils.h"
+#include "verilog/CST/verilog_nonterminals.h"
```

Comment:
I think these two headers can be moved to the .cc file and need not be exposed to includers of this header.